libs/sdk-model/__mocks__/model.ts

```diff
@@ -59,6 +59,63 @@ export const WinRate: IMeasure<IMeasureDefinition> = newMeasure("aaX0PIUzg7nF");
  * Metric Type: MAQL Metric
  */
 export const Won: IMeasure<IMeasureDefinition> = newMeasure("afSEwRwdbMeQ");
+
+/**
+ * Creates an adhoc measure with aggregation "sum"
+ */
+export const AggregateAdhoc = (): IMeasure<IMeasureDefinition> => {
+    const adhocMeasure = newMeasure("afSEwRwdbMeQ");
+    adhocMeasure.measure.definition.measureDefinition.aggregation = "sum";
+    return adhocMeasure;
+};
```

Comment:
a more 'native' approach to do this using model functions:

export const AggregateAdhoc = modifySimpleMeasure(Won, (m) => m.aggregation("sum").defaultLocalId())

Reply:
or alternatively, you don't even have to do this and use existing ad-hoc measure such as Duration.Sum

Reply:
Thanks, I've updated the PR.